cookbooks/aws-parallelcluster-install/attributes/default_redhat8.rb

```diff
@@ -10,7 +10,7 @@
                                              libical-devel postgresql-devel postgresql-server sendmail libxml2-devel libglvnd-devel
                                              mdadm python2 python2-pip libgcrypt-devel libevent-devel glibc-static bind-utils
                                              iproute NetworkManager-config-routing-rules python3 python3-pip iptables libcurl-devel yum-plugin-versionlock
-                                             coreutils moreutils sssd sssd-tools sssd-ldap curl environment-modules)
+                                             coreutils moreutils sssd sssd-tools sssd-ldap curl environment-modules gcc gcc-c++)
```

Comment:
@awslitvit @francesco-giordano I'm highlighting this since we discussed it off-line. We discussed the idea of installing the compiler for RHEL8 as a `stunnel` dependency (despite it arguably being conceptually incorrect and different from what we do, for example, in the case of other OSs [such as AL2](https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/cookbooks/aws-parallelcluster-install/attributes/default_amazon2.rb#L10)) and skip the installation of the base packages when we execute the test. However, then I verified that both for RHEL8 and for other OSs we also need other dependencies that we get when installing the base packages, so we cannot skip that phase.